src/services/teams/hooks/index.js

```diff
@@ -392,6 +392,9 @@ const hasTeamPermission = (permsissions, _teamId) => {
 			if (get(hook, 'isSuperhero') === true) {
 				return hook;
 			}
+			if (sessionUser.roles.map(r => r.name === 'student')) {
```

Comment:
- check get(hook... as in L392 for same functionality
- map does not seem to be the right method here maybe use array.includes() instead